### ISSUE TITLE
Two way databinding

### ISSUE THREAD

**Issue by IntelliJAbhishek** [VicMikhailau/MaskedEditText#29]
How to use this edit text with two way data binding?

**Comment by VicMikhailau** [comment]
Hi, @IntelliJAbhishek.
MskedEditText is just usual custom EditText. Please follow the android developer guidelines.
If you have some problems with MaskedEditText, you can create an own realization of it based on the code provided in the library.
Thanks.

**Comment by IntelliJAbhishek** [comment]
Yeah but due to text watcher we can't use data-binding directly

**Comment by VicMikhailau** [comment]
@IntelliJAbhishek Yes. I see. Unfortunately, I can't provide a particular way for that case. I can only suggest trying to use library classes directly in your project and override the needed behavior.

**Comment by VicMikhailau** [comment]
@IntelliJAbhishek If you find any solution please post it here and feel to free make any improvements to the library by creating e.g. a pull request.
Thanks.
